lms/djangoapps/lti_provider/views.py

```diff
@@ -70,11 +83,13 @@ def lti_launch(request, course_id, usage_id):
             params['oauth_consumer_key']
         )
     except LtiConsumer.DoesNotExist:
-        return HttpResponseForbidden()
+        template403 = Template(render_to_string('static_templates/403.html', {'return_url': return_url}))
+        return HttpResponseForbidden(template403.render())
```

Comment:
Before, on a launch error the page went blank- we need a "user friendly message" but I wasn't sure how much information we wanted to give the user. Should I add more to the message? And the return url is required by the certification as well when it's provided.